src/cloud/benchmark/query.py

```diff
@@ -118,3 +119,37 @@ def query(self, table: RemoteTable, **kwargs) -> Any:
             .limit(self.limit)
             .to_arrow()
         )
+
+
+class HybridQuery(Query):
+    """Simple hybrid search implementation"""
+
+    def __init__(
+        self,
+        words: List[str] = None,
+        metric: str = "cosine",
+        column: str = "text",
+        dim: int = 1536,
+        selected_columns: List[str] = None,
+        limit: int = 1,
+    ):
+        self.words = words or QUERY_WORDS
+        self.dim = dim
+        self.metric = metric
+        self.column = column
+        self.selected_columns = selected_columns or ["openai", "title"]
+        self.limit = limit
+
+    def query(self, table: RemoteTable, **kwargs) -> Any:
+        text_query = random.choice(self.words)
+        vector_query = np.random.standard_normal(self.dim)
+        reranker = RRFReranker()
+        return (
+            table.search(query_type="hybrid")
+            .vector(vector_query)
+            .text(text_query)
+            .rerank(reranker)
```

Comment:
will reranker adds latency?

Reply:
good question! I did test without the reranking step and the latency didn't change much. So I don't think reranker adds noticeable latency